CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-11592: Support to Get / Update / Mutate / Remove of groups via the `props` field and without the `props.id` field
   being set in the `/v1/groups` endpoint have been removed.
 - The unused "ComplianceRunSchedule" resource has been removed.
+- ROX-13034: Central reaches out to scanner `scanner.<namespace>.svc` now to respect OpenShift's `NO_PROXY` configuration.
```

Comment:
+1. Just be sure to add the training and docs labels to the Jira

Reply:
Good question. 
I assume the changelog is published to customers and is sufficient. 
+1 on the training label that support is aware.
What exactly does the docs label do?

Reply:
@gaurav-nelson can you share your opinion?

Reply:
We can mention it in the release notes and also add a note in the proxy configuration section. 

@SimonBaeumer related to @dhaus67's comment, if someone have `scanner.<namespace>` in their `NO_PROXY` settings:
- What issues they will face if they don't change anything?
- How can they fix those issues? I guess its by updating their `NO_PROXY` settings, so how to update those settings?
- Can they run a command to fix it? or do they have to manually change things?

Reply:
- Only if a `NO_PROXY` was configured to use all stackrox routes explicitly, and not configured to `*.svc`, scanner is not be reachable because the traffic will be routed through the proxy.
- Add `*.svc` or `scanner.<namespace>.svc` to the `NO_PROXY` setting 
- They can configure the NO_PROXY in the cluster-wide proxy settings in OpenShift here: https://docs.openshift.com/container-platform/4.11/networking/enable-cluster-wide-proxy.html

Reply:
@gaurav-nelson Are there any action items I should follow up with?